<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>732</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VideoPlayer / Ocean Coding School</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="90,5,0,5">
       <item>
        <widget class="QVideoWidget" name="view" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="bar">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="state">
           <property name="text">
            <string>Media Status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="playtime">
           <property name="text">
            <string>Running time</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_prev">
           <property name="text">
            <string>◀◀</string>
           </property>
           <property name="shortcut">
            <string>Left</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_play_pause">
           <property name="text">
            <string>▶ / ❚❚</string>
           </property>
           <property name="shortcut">
            <string>Space</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_forward">
           <property name="text">
            <string>▶▶</string>
           </property>
           <property name="shortcut">
            <string>Right</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_change">
           <property name="text">
            <string>Change Video</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_exit">
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
